scheduler/index.js

```diff
@@ -0,0 +1,116 @@
+const Bottleneck = require('bottleneck')
+
+const limiter = new Bottleneck({ maxConcurrent: 1, minTime: 0 })
+const ignoredAccounts = (process.env.IGNORED_ACCOUNTS || '')
+  .toLowerCase()
+  .split(',')
+
+const defaults = {
+  delay: !process.env.DISABLE_DELAY, // Should the first run be put on a random delay?
+  interval: 60 * 60 * 1000 // 1 hour
+}
+
+module.exports = (app, options) => {
```

Comment:
I recommend commenting here on forking the code with the link to the original repos and why:

1. original version no longer maintained.
1. needed compatibility with the latest version of probot.